remix/app/repositories/newsletter.server.test.ts

```diff
@@ -26,4 +26,32 @@ describe('Newsletter Repository', () => {
       );
     });
   });
+
+  describe('findOne', () => {
+    it('returns a newsletter', async () => {
+      const newsletter = { ...newsletterFactory };
+
+      prismaMock.newsletter.findUnique.mockResolvedValue(newsletter);
+
+      await expect(
+        NewsletterRepository.findOne({ where: { id: newsletter.id } })
+      ).resolves.toEqual(newsletter);
+    });
+  });
+
+  describe('update', () => {
+    it('updates a newsletter', async () => {
+      const result = { count: 1 };
```

Comment:
That variable is not really needed. Tests follow the AAA pattern (Arrange - Act - Assert) so since it will be used for assertion it should not be in the first block `Arrange`. 

Instead do the following as it is also more readable (no need to scroll back up 🙈 ):

```javascript
     await expect(
        NewsletterRepository.update({
          where: { id: newsletter.id },
          data: newsletter,
        })
      ).resolves.toEqual({ count: 1 });
```

Reply:
Well noted, Improved here: [7ad4f35](https://github.com/nimblehq/js-framework-benchmark/pull/126/commits/7ad4f356021b07ac686d227f05e4dc7e12b20884)